envoy.docs.sphinx_runner/setup.cfg

```diff
@@ -32,7 +32,7 @@ install_requires =
     docutils~=0.16.0
     envoy.base.runner
     envoy.base.utils
-    pygments==2.10.0
+    pygments>2.10.0
```

Comment:
previously this line itself was not there, and no version specification. Would that work?

Reply:
the reason i did it this way was because when we fix the bit of json in the docs, as has been done in the past, the new json might not be compatible with the old pygments version

i think this is the safest option

Reply:
so it turns out that pygments had a bug in version 2.11.0 - which we were hit by

upstream changelog: https://github.com/pygments/pygments/blob/master/CHANGES#L28

i think we need to pin this to `>= 2.11.1`

i have tested in netlify and 2.11.1 builds without issue

Reply:
dep bumped